src/Symfony/Component/Notifier/Bridge/Bluesky/BlueskyTransport.php

```diff
@@ -0,0 +1,227 @@
+            'json' => [
+                'identifier' => $this->user,
+                'password' => $this->password,
+            ],
+        ]);
+
+        try {
+            $statusCode = $response->getStatusCode();
+        } catch (TransportExceptionInterface $e) {
+            throw new TransportException('Could not reach the remote bluesky server.', $response, 0, $e);
+        }
+
+        if (200 !== $statusCode) {
+            throw new TransportException('Could not authenticate with the remote bluesky server.', $response);
+        }
+
+        try {
+            $this->authSession = $response->toArray(false) ?? [];
+        } catch (DecodingExceptionInterface $e) {
+            throw new TransportException('Unexpected response from bluesky server.', $response, 0, $e);
+        }
+    }
+
+    protected function doSend(MessageInterface $message): SentMessage
+    {
+        if (!$message instanceof ChatMessage) {
+            throw new UnsupportedMessageTypeException(__CLASS__, ChatMessage::class, $message);
+        }
+
+        if ([] === $this->authSession) {
+            $this->authenticate();
+        }
+
+        $post = [
+            '$type' => 'app.bsky.feed.post',
+            'text' => $message->getSubject(),
+            'createdAt' => (new \DateTimeImmutable())->format('Y-m-d\\TH:i:s.u\\Z'),
+        ];
+        if ([] !== $facets = $this->parseFacets($post['text'])) {
+            $post['facets'] = $facets;
+        }
+
+        $response = $this->client->request('POST', sprintf('https://%s/xrpc/com.atproto.repo.createRecord', $this->getEndpoint()), [
+            'auth_bearer' => $this->authSession['accessJwt'] ?? null,
+            'json' => [
+                'repo' => $this->authSession['did'] ?? null,
+                'collection' => 'app.bsky.feed.post',
+                'record' => $post,
+            ],
+        ]);
+
+        try {
+            $statusCode = $response->getStatusCode();
+        } catch (TransportExceptionInterface $e) {
+            throw new TransportException('Could not reach the remote bluesky server.', $response, 0, $e);
+        }
+
+        if (200 === $statusCode) {
+            $content = $response->toArray();
+            $sentMessage = new SentMessage($message, (string) $this);
+            $sentMessage->setMessageId($content['cid']);
+
+            return $sentMessage;
+        }
+
+        try {
+            $content = $response->toArray(false);
+        } catch (DecodingExceptionInterface $e) {
+            throw new TransportException('Unexpected response from bluesky server.', $response, 0, $e);
+        }
+
+        $title = $content['error'] ?? '';
+        $errorDescription = $content['message'] ?? '';
+
+        throw new TransportException(sprintf('Unable to send SMS with Bluesky: Status code %d (%s) with message "%s".', $statusCode, $title, $errorDescription), $response);
```

Comment:
```suggestion
        throw new TransportException(sprintf('Unable to send SMS with Bluesky: Status code %d (%s) with message "%s".', $statusCode, $title, $errorDescription), $response);
```
Is this really about SMS?

Reply:
Busted for copy-paste =)